### PR TITLE
php-mode: Adjust initialization of ggtags to avoid errors on startup

### PR DESCRIPTION
**Pull request by kostajh**
This fixes the start up error noted in #1140

### CODE REVIEW

File: contrib/lang/php/packages.el [hunk 1]
```diff
@@ -36,7 +36,8 @@
                                      #'ggtags-eldoc-function)))))
 
 (defun php/post-init-ggtags ()
-  (add-hook php-mode-hook 'ggtags-mode))
```
Comment (syl20bnr) [inline contrib/lang/php/packages.el]: The error is that `php-mode-hook` is not quoted.

**Comment by syl20bnr** [comment]
You don't need the condition because `post-init-ggtags` will be executed only if `init-ggtags` exist which means that the package is used.

The error is that `php-mode-hook` is not quoted.

**Comment by syl20bnr** [comment]
Thank you !
I modify your PR to include only the quote fix, tell me if this is OK on your side.
Cherry-picked in `develop` branch, you can safely delete your branch.

**Comment by kostajh** [comment]
thanks @syl20bnr!

**Pull request closed without merging**